tests/unit/Cli/RouterTest.php

```diff
@@ -0,0 +1,728 @@
+<?php
+
+namespace Phalcon\Test\Unit\Cli;
+
+use Phalcon\Test\Module\UnitTest;
+
+/**
+ * \Phalcon\Test\Unit\Cli\RouterTest
+ * Tests the \Phalcon\Cli\Router component
+ *
+ * @copyright (c) 2011-2016 Phalcon Team
+ * @link      http://www.phalconphp.com
+ * @author    Andres Gutierrez <andres@phalconphp.com>
+ * @author    Nikolaos Dimopoulos <nikos@phalconphp.com>
+ * @package   Phalcon\Test\Unit
+ *
+ * The contents of this file are subject to the New BSD License that is
+ * bundled with this package in the file docs/LICENSE.txt
+ *
+ * If you did not receive a copy of the license and are unable to obtain it
+ * through the world-wide-web, please send an email to license@phalconphp.com
+ * so that we can send you a copy immediately.
+ */
+class RouterTest extends UnitTest
+{
+    public function testRouters()
+    {
+        $this->specify(
+            "...",
+            function () {
+                $di = new \Phalcon\DI\FactoryDefault\CLI();
```

Comment:
1. Could you please use aliasing like `use Phalcon\Di\FactoryDefault\Cli?
2. Note `Di` not `DI` and `Cli` not `CLI`
3. Provide please real failed message  instead `"..."`